psalm.xml

```diff
@@ -42,6 +42,7 @@
 				<referencedClass name="OC\*" />
 				<referencedClass name="OC" />
 				<referencedClass name="OC\Security\CSP\ContentSecurityPolicyNonceManager" />
+				<referencedClass name="OCA\NotifyPush\Queue\IQueue" />
```

Comment:
You can add a copy of the Queue interface to tests/stub.phpstub so that psalm can also analyse it.